docs/INTEGRATIONS.rst

```diff
@@ -4,510 +4,436 @@ Integrations
 High Availability
 =================
 
-`Docker swarm <https://docs.docker.com/engine/swarm/>`__ is a
-clustering tool developped by Docker and ready to use with its
-containers. It allows to start a service, which we define and use as a
-means to ensure Zenko CloudServer's continuous availability to the end user.
-Indeed, a swarm defines a manager and n workers among n+1 servers. We
-will do a basic setup in this tutorial, with just 3 servers, which
-already provides a strong service resiliency, whilst remaining easy to
-do as an individual. We will use NFS through docker to share data and
+`Docker Swarm <https://docs.docker.com/engine/swarm/>`__ is a clustering tool
```

Comment:
Do we still test and support Docker Swarm?

Reply:
I'm going to ignore this for now, only because it exceeds the scope of the ticket. I believe you are 100% correct--we use Kubernetes do do this job now, so these instructions are in all likelihood wrong, or at least irrelevant. Unfortunately, I can't fix it by myself. 

Please open a ticket to this effect.